lib: otdoa_al: Allow user to set PLMN in CLI command

nriedel opened the pull request:
Adds optional MCC/MNC values to the phywi ecgi shell command.

nriedel committed:
lib: otdoa_al: Allow user to set PLMN in CLI command

Adds optional MCC/MNC values to the phywi ecgi shell command.

Signed-off-by: Neal Riedel <nriedel@serranosystems.com>

serranozack reviewed (changes requested):
https://github.com/PHY-Wireless/sdk-nrf-otdoa/blob/a9dadb77c073a9e8cb2c3e1b68ce8c67c61e0e04/lib/otdoa_al/otdoa_shell.c#L327

The number of command parameters needs to be increased, with only 2 the command will be rejected when the PLMN is given

nriedel committed:
lib: otdoa_al: Fix max number of arguments for phywi ecgi command

Fix the maximum number of arguments to be 5 to allow support for
MCC/MNC in phywi ecgi command.

Fix some compliance issues.

Signed-off-by: Neal Riedel <nriedel@serranosystems.com>
